src/database/sqlite.rs

```diff
@@ -467,7 +467,7 @@ impl SqliteDatabase {
         &self,
         txid: &[u8],
     ) -> Result<Option<TransactionDetails>, Error> {
-        let mut statement = self.connection.prepare_cached("SELECT transaction_details.timestamp, transaction_details.received, transaction_details.sent, transaction_details.fee, transaction_details.height, transactions.raw_tx FROM transaction_details, transactions WHERE transaction_details.txid=transactions.txid AND transaction_details.txid=:txid")?;
+        let mut statement = self.connection.prepare_cached("SELECT timestamp, received, sent, fee, height FROM transaction_details WHERE txid=:txid")?;
```

Comment:
I'd modify the commit message of 27c6735d122ea7ef61f19e1eeac23643ccc096e8: 
The main commit line shouldn't be "Fix failing test: test_del_tx" : the purpose of this commit is to fix a bug you've found, not to fix a test (the commit doesn't even have lines changing tests, so it's definitely not fixing a test) :)